guides/developers/event-display.md

```diff
@@ -23,7 +24,7 @@ The modular and extendable nature of Phoenix is not limited to just the `EventDi
 
 ## [`ThreeManager`](../../packages/phoenix-event-display/src/managers/three-manager/index.ts)
 
-The `ThreeManager` is responsible for performing all `three.js` related functions. Any feature that depends on the `three.js` library is implemented through the `ThreeManager`. Since Phoenix has a lot of features implemented through `three.js`, the `ThreeManager` is further divided into sub managers which have different roles.
+The `ThreeManager` is responsible for performing all `three.js` related functions. Any feature that depends on the `three.js` library is implemented through the `ThreeManager`. Since, Phoenix has a lot of features implemented through `three.js`, the `ThreeManager` is further divided into sub managers which have different roles.
```

Comment:
I think the added `,` is not needed here.

Reply:
🤦